src/source.cc

```diff
@@ -88,9 +88,83 @@ std::string Source::FixIt::string(Glib::RefPtr<Gtk::TextBuffer> buffer) {
 const std::regex Source::View::bracket_regex("^([ \\t]*).*\\{ *$");
 const std::regex Source::View::no_bracket_statement_regex("^([ \\t]*)(if|for|else if|while) *\\(.*[^;}] *$");
 const std::regex Source::View::no_bracket_no_para_statement_regex("^([ \\t]*)(else) *$");
+const Glib::ustring right_click_menu_xml= R"RAW(<interface>
+    <menu id='right-click-line-menu'>
+      <section>
+        <item>
+          <attribute name='label' translatable='yes'>_Undo</attribute>
+          <attribute name='action'>app.edit_undo</attribute>
+        </item>
+        <item>
+          <attribute name='label' translatable='yes'>_Redo</attribute>
+          <attribute name='action'>app.edit_redo</attribute>
+        </item>
+      </section>
+      <section>
+        <item>
+          <attribute name='label' translatable='yes'>_Paste</attribute>
+          <attribute name='action'>app.edit_paste</attribute>
+        </item>
+      </section>
+      <section>
+        <item>
+          <attribute name='label' translatable='yes'>_Toggle _Line _Comment</attribute>
+          <attribute name='action'>app.source_comments_toggle</attribute>
+        </item>
+        <item>
+          <attribute name='label' translatable='yes'>_Rename</attribute>
+          <attribute name='action'>app.rename</attribute>
+        </item>
+      </section>
+    </menu>
+    <menu id='right-click-selected-menu'>
+      <section>
+        <item>
+          <attribute name='label' translatable='yes'>_Undo</attribute>
+          <attribute name='action'>app.edit_undo</attribute>
+        </item>
+        <item>
+          <attribute name='label' translatable='yes'>_Redo</attribute>
+          <attribute name='action'>app.edit_redo</attribute>
+        </item>
+      </section>
+      <section>
+        <item>
+          <attribute name='label' translatable='yes'>_Cut</attribute>
+          <attribute name='action'>app.edit_cut</attribute>
+        </item>
+        <item>
+          <attribute name='label' translatable='yes'>_Copy</attribute>
+          <attribute name='action'>app.edit_copy</attribute>
+        </item>
+        <item>
+          <attribute name='label' translatable='yes'>_Paste</attribute>
+          <attribute name='action'>app.edit_paste</attribute>
+        </item>
+      </section>
+      <section>
+        <item>
+          <attribute name='label' translatable='yes'>_Toggle _Line _Comment</attribute>
+          <attribute name='action'>app.source_comments_toggle</attribute>
+        </item>
+      </section>
+    </menu>
+  </interface>)RAW";
 
 Source::View::View(const boost::filesystem::path &file_path, Glib::RefPtr<Gsv::Language> language): Gsv::View(), SpellCheckView(), DiffView(file_path), language(language), status_diagnostics(0, 0, 0) {
   load();
+  if(!right_click_line_menu || !right_click_selected_menu){
+    auto builder = Gtk::Builder::create_from_string(right_click_menu_xml);
+    const auto create_menu = [=](const std::string &menu){
```

Comment:
Use `[this, builder]` here instead of `[=]`. `[=]` creates copies of every variable available. This might be the cause of the Fedora test fails.